requirements.txt

```diff
@@ -1,4 +1,6 @@
 # Dependencies required to build and test c-lightning
+
+mistune ~= 0.8.4
```

Comment:
This is an indirect dependence, you can solve your issue with the following command `pip3 install -r requirements.lock`

This is a leak of the mrkd https://github.com/refi64/mrkd/issues/7 

For now, we are using the option to install mistune by hand with the command `pip3 install mistune == 0.8.4`

Reply:
Fixed in f238cdcbcaf426af15d25678c3fa5f0c7e2c6068